src/main/java/pl/agh/edu/iisg/io/vmms/vmmsbackend/model/VMPool.java

```diff
@@ -0,0 +1,95 @@
+package pl.agh.edu.iisg.io.vmms.vmmsbackend.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.model.reservations.Reservation;
+
+import javax.persistence.*;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name = "vm_pools")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+public class VMPool {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @Column(unique=true, nullable=false)
+    private String shortName;
+
+    @NotNull
+    private String displayName;
+
+    @Min(0)
+    @NotNull
+    private Integer maximumCount;
+
+    @NotNull
+    private Boolean enabled;
+
+    private String description;
+
+    @OneToMany(mappedBy = "pool")
+    public Set<Reservation> reservations;
+
+    public abstract static class Builder{
```

Comment:
Jesteś pewna że potrzebujemy buildera? Wydaje mi się że adnotacje @Data lomboka robią swoje, dzięki temu całkowicie rezygnujemy z getterów i setterów i objętość klasy znacznie się redukuje

Reply:
Dziękuję : ), nie znałam lomboka